native/acter/src/api/device.rs

```diff
@@ -66,15 +66,20 @@ impl DeviceChangedEvent {
 
 #[derive(Clone, Debug)]
 pub(crate) struct DeviceController {
+    new_event_tx: Sender<DeviceNewEvent>, // keep it resident in memory
```

Comment:
I don't think we prompt anymore for new-device detections (as they were very buggy and reported even the device I was just logging myself in-with). Not sure this features is still needed or shouldn't be completely removed then.

Reply:
Now detection of new device works fine.
`allSessionsProvider` is updated on every detection.

![image](https://github.com/user-attachments/assets/cc07a538-e490-4f61-bb40-6487c53d5539)

Do you want that acter would pop up dialog like this?

![image](https://github.com/user-attachments/assets/4f808d9c-e432-4c9c-811a-e45d1006ea88)